sample.env

```diff
@@ -9,4 +9,16 @@ VITE_TRANSITIVE_USER=
 
 # The JWT secret of the above user on the above Transitive deployment's
 # Security page
-JWT_SECRET=
\ No newline at end of file
+JWT_SECRET=
+
+# Secret used by express-session
+TR_SESSION_SECRET=SOME_STRONG_SECRET_LIKE_CH9UwdjGck4DKGMbftVbBm4r9SuZQKDP
```

Comment:
Let's not use the `TR_` prefix here. That's short for Transitive Robotics and we are using it for Transitive itself, so let's not overload it. Maybe we can use `TRANSACT_`?